rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -1399,6 +1399,32 @@ def get_responses_to_surveys(
 
         return dict(participant_response_map)
 
+    @classmethod
+    def get_latest_answer_for_state_receiving_care(cls, session: Session, participant_id) -> str:
+        answer_code = aliased(Code)
+        question_code = aliased(Code)
+        query = (
+            session.query(answer_code.value)
+            .select_from(QuestionnaireResponse)
+            .join(QuestionnaireResponseAnswer)
+            .join(QuestionnaireQuestion)
+            .join(
+                question_code,
+                and_(
+                    question_code.codeId == QuestionnaireQuestion.codeId,
+                    question_code.value == code_constants.RECEIVE_CARE_STATE
+                )
+            )
+            .join(
+                answer_code,
+                answer_code.codeId == QuestionnaireResponseAnswer.valueCodeId
+            )
+            .order_by(QuestionnaireResponse.authored.desc())
```

Comment:
This is an admittedly small risk, but:  we have gotten two payloads for a module (TheBasics) with an identical authored time but conflicting non-null answer codes to the same question, so using only authored for the `order_by` can have indeterminate results.

For PDR we often use `QuestionnaireResponse.created.desc() ` as a secondary `order_by ` value just for predictable ordering results.